winml/test/api/raw/winml_microsoft.h

```diff
@@ -262,7 +262,10 @@ class WinMLLearningModel
 
     int32_t Initialize(const char* bytes, size_t size)
     {
-        RoInitialize(RO_INIT_TYPE::RO_INIT_SINGLETHREADED);
+      auto hr = RoInitialize(RO_INIT_TYPE::RO_INIT_SINGLETHREADED);
+      if (hr < 0 && hr != RPC_E_CHANGED_MODE) {
```

Comment:
can you add a comment about what RPC_E_CHANGED_MODE is and why it's a special case here?